ts/private/build_test.bzl

```diff
@@ -26,11 +29,22 @@ def _build_test_impl(ctx):
         runfiles = ctx.runfiles(files = ctx.files.targets),
     )
 
-build_test = rule(
+_build_test = rule(
     doc = _DOC,
     implementation = _build_test_impl,
     attrs = {
         "targets": attr.label_list(allow_files = True),
+        "is_windows": attr.bool(mandatory = True),
     },
     test = True,
 )
+
+def build_test(name, **kwargs):
+    _build_test(
+        name = name,
+        is_windows = select({
+            "@bazel_tools//src/conditions:host_windows": True,
```

Comment:
shouldn't this depend on the exec platform?

Reply:
I have no idea, I just copied it and it made Windows ci green 🤷‍♂️

Reply:
Well if anyone tries to use windows host with Linux RBE I guess they will hit it :)